ducktape/tests/test.py

```diff
@@ -52,12 +54,22 @@ def min_cluster_size(self):
         """
         return self.test_context.services.num_nodes()
 
-    def setUp(self):
+    def setup(self):
```

Comment:
Why are we making this change? I thought we had discussed previously the noun vs verb usage? Does anyone besides py.test use the noun form? I know jUnit and Python's unittest module both use the verb form.

And if we are going this path, then a [deprecation warning](https://docs.python.org/2/library/warnings.html) is warranted. Otherwise people have to dig into the code to understand what they are supposed to be overriding.